README.md

```diff
@@ -47,6 +47,48 @@ As is the case with Drupal itself, in Organic Groups different permissions can
 be assigned to different user roles. This allows group members to perform a
 different set of actions, in different group contexts.
 
+### OG Membership Entity
+
+The membership entity that connects a group and a user.
```

Comment:
@damiankloip maybe it worth for someone that actually knows English to go over this :)